.github/workflows/build_integration_and_code_analysis.yml

```diff
@@ -0,0 +1,73 @@
+name: Integration and code analysis
+on:
+  workflow_dispatch: null
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+    schedule:
+      - cron: '* 5 * * *'
+jobs:
+  Sonar-Cloud:
+    name: Static Code Analysis - Sonar Cloud
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK 17
+        uses: actions/setup-java@v1
+        with:
+          java-version: 17
```

Comment:
does it support our specific version ? :
java-version: '17'
distribution: 'corretto'
cache: 'maven'

If so I think it would be better to add it.

Reply:
I think it should, I'll try it.